src/update-all/index.ts

```diff
@@ -1,6 +1,8 @@
-const preload = () => import("./update-all-dialog");
+const preload = () => import("./update-all-process-dialog");
```

Comment:
let's revert this rename as the folder has not changed either.

Reply:
done